### PR TITLE
Pattern cypress

### PR DESCRIPTION
**Pull request by moran711**

### CODE REVIEW

File: cypress/integration/business-pages/business-pages.test.js [hunk 1]
```diff
@@ -22,7 +23,8 @@ describe('Business pages test ', () => {
     cy.stubRequest('getAllBusinessTexts', getAllBusinessTexts).as(
       'getAllBusinessTexts'
     );
-    cy.visit('/business-pages');
+    cy.visit('/');
```
Comment (kapoliub) [inline cypress/integration/business-pages/business-pages.test.js:26]: maybe make sense to receive path from configs

File: cypress/integration/pattern/pattern.test.js [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+} from './pattern.variables';
+import statuses from '../../../src/configs/statuses';
+import routes from '../../../src/configs/routes';
+import { patternTranslations } from '../../../src/translations/pattern.translations';
+
+describe('pattern tests', () => {
+  beforeEach(() => {
+    cy.login(Cypress.env('ADMIN_LOGIN'), Cypress.env('ADMIN_PASSWORD'));
+    cy.stubRequest('getAllPatterns', getAllPatterns).as('getAllPatterns');
+    cy.visit('/');
```
Comment (kapoliub) [inline cypress/integration/pattern/pattern.test.js:20]: same

File: cypress/integration/pattern/pattern.test.js [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+    cy.get('[data-cy=en-description]').type('a');
+    cy.get('[data-cy=en-description-error]').contains(
+      config.patternErrorMessages.PATTERN_VALIDATION_ERROR
+    );
+    cy.get('[data-cy=ua-tab]').click();
+    cy.get('[data-cy=ua-name]').type('a');
+    cy.get('[data-cy=ua-name-error]').contains(
+      config.patternErrorMessages.PATTERN_VALIDATION_ERROR
+    );
+    cy.get('[data-cy=ua-description]').type('a');
```
Comment (kapoliub) [inline cypress/integration/pattern/pattern.test.js:105]: maybe it makes sense to move 'a' value to variable

**Review by ivan-bonk — APPROVED** [review]

**Review by kapoliub — APPROVED** [review]

File: cypress/integration/pattern/pattern.test.js (hunk 3 above)
Comment (moran711) [reply cypress/integration/pattern/pattern.test.js]: fixed

File: cypress/integration/business-pages/business-pages.test.js (hunk 1 above)
Comment (moran711) [reply cypress/integration/business-pages/business-pages.test.js:26]: We don't have home page on '/' route. When we merge pr with homepage I will fix it

File: cypress/integration/pattern/pattern.test.js (hunk 2 above)
Comment (moran711) [reply cypress/integration/pattern/pattern.test.js:21]: We don't have home page on '/' route. When we merge pr with homepage I will fix it

File: cypress/integration/pattern/pattern.test.js [hunk 4]
```diff
@@ -0,0 +1,175 @@
...
+    cy.get('[data-cy="Код матеріалу"]').contains(
+      config.tableHeadRowTitles.patterns[2]
+    );
+    cy.get('[data-cy=Доступний]').should('be.visible');
+    cy.get('[data-cy=Доступний]').contains(
+      config.tableHeadRowTitles.patterns[3]
+    );
+    cy.get('[data-cy=Дії]').should('be.visible');
+    cy.get('[data-cy=Дії]').contains(config.tableHeadRowTitles.patterns[4]);
+
```
Comment (koropalov) [inline cypress/integration/pattern/pattern.test.js:46]: I am not sure, but maybe it would be better, to write selectors by using Latin letters.

**Review by koropalov — APPROVED** [review]

File: cypress/integration/pattern/pattern.test.js (hunk 4 above)
Comment (moran711) [reply cypress/integration/pattern/pattern.test.js:46]: We dynamically pass this values from our config file.

**Pull request merged**